### PR TITLE
1.2.0

### PR DESCRIPTION
**Pull request by elharo**
@parkjam4 @briandealwis  I want to push a beta version of this for testing.

### CODE REVIEW

File: gcp-repo/metadata.product [hunk 1]
```diff
@@ -4,7 +4,7 @@
   This product exists to (1) bring in any required features, and
   (2) add any additional metadata for the CT4E repository.
 -->
-<product uid="com.google.cloud.tools.eclipse.dist" version="1.1.0.qualifier" useFeatures="true" includeLaunchers="false">
```
Comment (briandealwis) [inline gcp-repo/metadata.product]: You need this change, though it doesn't hurt: it's not really visible unless people start examining the p2 metadata.

Comment (elharo) [inline gcp-repo/metadata.product]: Did you mean to say you don't need this change?

Comment (briandealwis) [inline gcp-repo/metadata.product]: Oops, that's right — *don't* need this change.

File: gcp-repo/metadata.p2.inf [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 requires.1.namespace=org.eclipse.equinox.p2.iu
 requires.1.name=com.google.cloud.tools.eclipse.category
-requires.1.range=[1.0.0,1.1.0]
+requires.1.range=[1.0.0,1.2.0]
 requires.1.greedy=true
 
 units.1.id=com.google.cloud.tools.eclipse.category
```
Comment (chanseokoh) [inline gcp-repo/metadata.p2.inf]: The line right below is units.1.version=1.1.0. Its value should have been 1.2.0.

**Pull request merged**